TesseractOCR/G8Tesseract.h

```diff
@@ -260,18 +275,26 @@ extern NSInteger const kG8MaxCredibleResolution;
  */
 @property (nonatomic, weak) id<G8TesseractDelegate> delegate;
 
-/// The default initializer should not be used since the language Tesseract
-/// uses needs to be explicit.
-- (instancetype)init __attribute__((unavailable("Use initWithLanguage:language instead")));
+/**
+ *  The default initializer.
+ *  @return A G8Tesseract instance initialized with niether language,
+ *          nor any other settings specified. You can set the language later, 
+ *          using language property.
+ */
+- (instancetype)init;
 
 /**
  *  Initialize Tesseract with the provided language.
  *
  *  @param language The language to use in recognition. See `language`.
  *
- *  @return The initialized Tesseract object, or `nil` if there was an error.
+ *  @return The initialized Tesseract object. 
+ *
+ *  @note   You should check that the language property match the languages 
+ *          you have specified. Also you may ensure that tesseract is properly
```

Comment:
It looks like the convention we've been using is to capitalize "Tesseract", so can you change any instances of "tesseract" in these files to "Tesseract"?